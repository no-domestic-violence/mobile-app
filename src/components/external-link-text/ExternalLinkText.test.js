import React from 'react';
import ExternalLinkText from './index';
import { render, fireEvent } from '@testing-library/react-native';

describe('External Link Text component', () => {
  const url = 'www.help.de';
  const title = 'www.help.de';
  const openExternalLink = jest.fn();
  it('renders component', () => {
    const { getByText } = render(<ExternalLinkText title={title} url={url} />);
    const text = getByText('www.help.de');
    expect(text).not.toBeNull();
    // TODO: to test on press when design is ready
  });

  it('should match snapshot', () => {
    const result = render(
      <ExternalLinkText title={title} url={url} />
    ).toJSON();
    expect(result).toMatchSnapshot();
  });
});
