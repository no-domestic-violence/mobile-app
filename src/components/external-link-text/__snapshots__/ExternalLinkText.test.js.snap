// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`External Link Text component should match snapshot 1`] = `
<Text
  onPress={[Function]}
  style={
    Object {
      "color": "#242424",
    }
  }
  type="clear"
>
  www.help.de
</Text>
`;
